tools/cldr/build.xml

```diff
@@ -0,0 +1,65 @@
+<!-- © 2020 and later: Unicode, Inc. and others.
+     License & terms of use: http://www.unicode.org/copyright.html -->
+
+<!-- This build file is intended to become the single mechanism for working with CLDR
+     code and data when building ICU data.
+     
+     Eventually it will encompass:
+     * Building ICU data form CLDR data via cldr-to-icu.
+     * Building the CLDR libraries needed to support ICU data conversion.
+     * Copying CLDR test data for ICU regression tests.
+
+     It's not complete yet, so for now follow the instructions in:
+       icu4c/source/data/cldr-icu-readme.txt
+     -->
+
+<!-- Using the directory of this build.xml file as ${basedir}. -->
+<project name="CLDR" default="install" basedir=".">
+
+    <!-- Installs prerequisite data from CLDR (test data and CLDR libraries).
+         This target will not rebuild the ICU data however (since that's a
+         lengthy process). -->
+    <target name="install" depends="clean, copy-cldr-testdata, install-cldr-libs"/>
+
+    <!-- Cleans CLDR dependencies (testdata, CLDR libraries). This target does
+         not delete any generated ICU data. -->
+    <target name="clean" depends="clean-cldr-testdata"/>
+
+    <!-- Initialize any properties not already set on the command line. -->
+    <target name="init-args">
+        <property environment="env"/>
+        <!-- Inherit properties from environment variable unless specified. As usual
+             with Ant, this is messier than it should be. All we are saying here is:
+             "Use the property if explicitly set, otherwise use the environment variable."
+             We cannot just set the property to the environment variable, since expansion
+             fails for non existant properties, and you are left with a literal value of
+             "${env.CLDR_DIR}". -->
+        <condition property="cldrDir" value="${env.CLDR_DIR}">
+            <isset property="env.CLDR_DIR"/>
+        </condition>
+        <fail unless="cldrDir"
+             message="Set the CLDR_DIR environment variable (or cldrDir property) to the top level CLDR source directory (containing 'common')."/>
+    </target>
+
+    <!-- TODO: Add properties and targets for copying test data. -->
```

Comment:
Adding infrastructure to copy the CLDR test data is the original purpose of Hugo's PR.  Please include it here, with the new paths discussed previously.

Reply:
Why? Hugo has said he'll do the work when this is in. I literally have no idea, other than copying Hugo's original file what's needed. You're saying it's better if we delay this PR by days to so I can ask Hugo to tell me what to add, rather than getting this in and having Hugo add his stuff in a follow up PR?

Reply:
Or, you can clone this and do whatever you want to do on top of it (since you know what needs doing and I don't).
Either way I have no interest in becoming someone who needs to negotiate between you and Hugo to decide what to add here, that's just not a productive use of my time.

Reply:
Dealing with this in a follow-up review. (And trying to figure out how to mark a PR as "reviewed" by me after it's already been merged.)